services/core-web/common/actionCreators/noticeOfWorkActionCreator.js

```diff
@@ -168,7 +168,7 @@ export const fetchOriginalNoticeOfWorkApplication = (applicationGuid) => (dispat
 export const updateNoticeOfWorkApplication = (
   payload,
   nowApplicationGuid,
-  message = "Successfully updated Notice of Work"
+  message = "Successfully updated the permit application."
```

Comment:
Did we try to make this generic here because we don't know if they are updating an AA or NoW?

Reply:
It might be worth passing in the application_type_code, that way we can display a customized toast message.

Reply:
I was thinking of doing that, but didn't know if it was worth the effort. Thoughts?

Reply:
Exactly.